xml/System/Span`1+Enumerator.xml

```diff
@@ -21,8 +21,23 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides an enumerator for the elements of a <see cref="T:System.Span`1" />.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+The C# [foreach](~/docs/csharp/language-reference/keywords/foreach-in.md) of the C# language and the [For Each...Next](~/docs/visual-basic/language-reference/statements/for-each-next-statement.md) construct in Visual Basic hides the complexity of enumerators. Instead of directly manipulating the enumerator, using `foreach` or `For Each...Next` is recommended. 
```

Comment:
It may be worth mentioning these two points:

`Span<T>.Enumerator` does not implement `IEnumarator<T>`.
`Span<T>.Enumerator` does not have a `Reset()` method.

Reply:
@BillWagner, thanks. These are definitely worth mentioning.